cluster-up-example

```diff
@@ -1,14 +1,10 @@
 [Nodes:children]
 master
-core
-mbaas
+macos
```

Comment:
I think it would be better to remove the `ansible_connection=local` rather than removing the macos node from the list. that way the macos host has access to all the node variables. As we discussed yesterday we only need to do ansible_connection=local in the case where we don't have ssh access to the nodes. i.e dedicated